modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -324,6 +381,38 @@ NOTICE,user2,user2@example.com,"2024/04/03 15:20:32
 
 As you can see above, there is `Duplicate` notice for a file and another one for a directory name. `Duplicate` notices are logged for case conflicts. A conflict takes place because a file or directory that has been migrated earlier is in conflict with the name of the reported object. The conflicts for the particular users need to be resolved within ownCloud. When this is done, the migration can be restarted. `rclone` will compare both sides to identify already migrated objects and will continue with those objects that have not been migrated yet.
 
+==== Migrate Shares
+
+IMPORTANT: When running this command, you cant run the `migrate:files|users` command anymore!
+
+Issue the following command to start migrating shares:
+
+[source,bash]
+----
+sudo -u www-data \
+  php /var/www/owncloud/occ \
+  migrate:to-kiteworks:shares \
+  $KW_ADMIN_USER
+----
+
+The command does not require user interaction. It can be run e.g. as a screen session so that reported issues can be seen directly or as a background job.
```

Comment:
this command creates a migrate-kiteworks-shares-<timestamp>.csv

Reply:
fixed